src/foam/nanos/crunch/AgentCapabilityJunctionDAOSummaryView.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'AgentCapabilityJunctionDAOSummaryView',
```

Comment:
This view is specific to Approvals and as such should be in the approval package. 
Also there is reference to a nanopay classification.

Reply:
The Approvals package could contain a refinement to AgentCapabilityJunction where it includes the DAOSummaryView as an inner-class. Then all of the code is put in the correct place but can remain on the nanopay side.